OpenRA.Mods.Common/Terrain/DefaultTerrain.cs

```diff
@@ -0,0 +1,194 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2020 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using OpenRA.FileSystem;
+using OpenRA.Primitives;
+using OpenRA.Support;
+
+namespace OpenRA.Mods.Common.Terrain
+{
+	public class DefaultTerrainLoader : ITerrainLoader
+	{
+		public DefaultTerrainLoader(ModData modData) { }
```

Comment:
Not used?

Reply:
Used by reflection:

https://github.com/OpenRA/OpenRA/blob/2d6e3062278d5bdd726da24f16144dbe6df361fc/OpenRA.Game/ModData.cs#L77-L82